.github/workflows/ci.yml

```diff
@@ -182,7 +182,7 @@ jobs:
         run: |
           whereis git
           git fetch origin ${{ github.base_ref }}
```

Comment:
Remove `git fetch` since it's not used by the script

Reply:
done